### DISCUSSION TITLE
how to use the .config file?

### DISCUSSION THREAD

**Discussion by JM-Bader** [MannLabs/alphadia#112] [Ideas]
How to re-load a config file to load the settings?

**Comment by mschwoer** [comment]
this is currently not supported, we will put in on the backlog

**Reply by mschwoer** [reply]
internal: https://github.com/orgs/MannLabs/projects/12/views/1?filterQuery=config+load&pane=issue&itemId=111013063

**Comment by mschwoer** [comment]
Fixed a while ago (frozen_config.yaml), documented in 1.12.2
